protocol/v2/qbft/storage/ibft_store.go

```diff
@@ -2,14 +2,18 @@ package qbftstorage
 
 import (
 	"encoding/json"
+	"fmt"
+	"math/bits"
 
 	"github.com/attestantio/go-eth2-client/spec/phase0"
-
 	specqbft "github.com/ssvlabs/ssv-spec/qbft"
 	spectypes "github.com/ssvlabs/ssv-spec/types"
+
 	"github.com/ssvlabs/ssv/exporter/convert"
 )
 
+const maxCommitteeSize = 13 // TODO: define in ssv network config
+
 // StoredInstance contains instance state alongside with a decided message (aggregated commits).
 type StoredInstance struct {
```

Comment:
Not exactly relevant to this PR, but I thought we no longer use this struct after https://github.com/ssvlabs/ssv/pull/1767 (except for in related tests, but these should also be adjusted accordingly then) ? Perhaps can remove while at it.

Reply:
It's used in `TestHandleDecidedQuery` but honestly I didn't check if we need it. Let's leave this for another PR as this one is a hotfix and we need to merge it ASAP

Reply:
@nkryuchkov @Tom-ssvlabs lets make sure we have ticket -

> removing unused instance storage structs (`protocol/v2/qbft/storage/ibft_store.go`)

Reply:
https://blox-ensemble.monday.com/boards/7946005181/pulses/7957409842